src/main/resources/api/swagger.yaml

```diff
@@ -44,73 +76,73 @@ paths:
     post:
       summary: Creates a new feature
       description: |
-        Creates a new feature with specified name, docker data, etc. Feature 
+        Creates a new feature with specified name, docker data, etc. Feature
         name is checked to be unique.
       parameters:
         - name: feature
           required: true
           in: body
           schema:
-            $ref: '#/definitions/FeatureToCreate'
+            $ref: '#/definitions/Feature'
       tags:
         - Features
       responses:
         201:
           description: Feature successfully created
           schema:
-            $ref: '#/definitions/FeatureFull'
+            $ref: '#/definitions/Feature'
         default:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-  /api/features/{id}:
+  /api/features/{name}:
     parameters:
-      - name: id
-        in: path
-        required: true
-        description: Name of feature
-        type: string
+    - name: name
+      in: path
+      description: the feature name.
+      required: true
+      type: string
     get:
-      summary: Get full feature elements
+      summary: Get single feature
       description: |
-        Returns all the feature information, including dependencies
+        Returns all the feature data without it's dependencies.
       tags:
         - Features
       responses:
         200:
-          description: Full feature representation
+          description: a singel feature
           schema:
-            $ref: '#/definitions/FeatureFull'
+            $ref: '#/definitions/Feature'
         default:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
     put:
       summary: Replace this feature's content
       description: |
-        Replaces docker_data and dependencies for this feature. Recursion checking is performed on this step
-      tags: 
+        Replaces snippet, test_snippet and meta data for this feature.
+      tags:
         - Features
       parameters:
         - name: feature
           required: true
           in: body
           schema:
-            $ref: '#/definitions/FeatureToCreate'
+            $ref: '#/definitions/Feature'
       responses:
         200:
-          description: Full feature representation
+          description: Feature successfully updated
           schema:
-            $ref: "#/definitions/FeatureFull"
+            $ref: "#/definitions/Feature"
         default:
           description: Unexpected exception
           schema:
             $ref: "#/definitions/Error"
     delete:
       summary: Delete feature
       description: |
-        Delete this feature. In case when another features has links to this feature - nothing will be deleted and
-        error will be returned.
+        Delete this feature. If another features has links to this feature
```

Comment:
This phrase is misconstructed:
"If other features have links to this feature"
or
"If another feature has links to this feature"